planetarium/NineChronicles
# Feature/apply revomon to lobby

### Description

1. Add event types: BeforeRevomon, Revomon, AfterRevomon
2. Configure event data
   - BeforeRevomon: 01/01 00:00:00 ~ 04/04 14:59:59
   - Revomon: 04/04 15:00:00 ~ 04/19 14:59:59
   - AfterRevomon: 04/19 15:00:00 ~ 12/31 23:59:59
3. Apply revomon character to the lobby
4. Open url when click the revomon in the lobby

### How to test

1. Checkout the `feature/apply-revomon-to-lobby` branch.
2. Configure revomon event data in the `EventData` scriptable object.
3. Play the game and check the lobby.

### Related Links

1. [revomon 캐릭터 메인로비 표시](https://app.asana.com/0/958521740385861/1202022984349854/f)
2. [revomon 메인화면 링크 연결 ](https://app.asana.com/0/958521740385861/1202022910775680/f)

### Screenshot

<img width="163" alt="image" src="https://user-images.githubusercontent.com/6128868/160540522-86ece58f-7a9d-4346-82da-704b1251eda6.png">
